index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="scss/application.css">
+</head>
+<body>
+  <h1 class="title">Hello World</h1> 
+</body>
+</html>
```

Comment:
add an empty line at the end of file, you can configure this in your text editor to add it automatically